pkg/catalog/types.go

```diff
@@ -93,6 +90,12 @@ type MeshCataloger interface {
 
 	// ListInboundTrafficTargetsWithRoutes returns a list traffic target objects composed of its routes for the given destination service identity
 	ListInboundTrafficTargetsWithRoutes(identity.ServiceIdentity) ([]trafficpolicy.TrafficTargetWithRoutes, error)
+
+	// GetWeightedClustersForUpstream lists the apex services from traffic split policies for a given upstream.
```

Comment:
This comment needs to be updated to reflect what is being returned, ie. weighted cluster backends corresponding to the upstream service.

Reply:
Thanks, fixed.